test/services/event_service_test.exs

```diff
@@ -1,16 +1,16 @@
 defmodule Aprb.Service.EventServiceTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
```

Comment:
Why does this have to change? Does this force your summary task to run before it returns?